packages/clay-form/src/SelectBox.tsx

```diff
@@ -96,11 +95,6 @@ interface IProps extends React.HTMLAttributes<HTMLDivElement> {
 	 */
 	onSelectChange: (val: Array<string>) => void;
 
-	/**
-	 * Id of the selectbox to poperly link it to the label
-	 */
-	selectBoxId?: string;
```

Comment:
Since the component itself is called a `Select`, I think its important we extend from its html element so that it behaves similarly to how people would use the html select. By doing this, we only need the `id`